style.md

```diff
@@ -1802,16 +1802,16 @@ if f, err := os.Open("f"); err == nil {
 </td><td>
 
 ```go
-f, err := os.Open("f")
+data, err := ioutil.ReadFile(name)
 if err != nil {
    return err
 }
 
-if _, err := io.WriteString(f, "data"); err != nil {
+if err := cfg.Decode(data); err != nil {
   return err
 }
 
-return f.Close()
+return nil
```

Comment:
maybe we have a comment like
```
// use cfg for some operations

return nil
```

otherwise, this would probalby just be `return cfg.Decode(data)`

Reply:
👍 going to print it as a placeholder for "Do stuff with cfg"